src/assets/common.css

```diff
@@ -75,3 +75,17 @@ dl.variant-info-middle dd {
     max-height: 400px;
     overflow-x: hidden;
 }
+
+.tabulator .tabulator-header .tabulator-col .tabulator-col-content .tabulator-col-title {
+    white-space: normal;
+    text-overflow: fade;
+  }
+
+  /* Prevent Bootstrap buttons from covering LZ tooltips, by giving render preference to tooltips */
+  .lz-data_layer-tooltip {
+    z-index: 2;
+  }
+
+  input.form-control {
```

Comment:
This rule in `common.css` would change the definition of all bootstrap form-controls, everywhere on every page in the site, including any that might be added in the future.

If this is aimed at the search box, might I suggest a more targeted CSS selector? The single-file-component approach also means that styles specific to the search box can be kept in the `SearchBox.vue` component. 

(which makes it easier to clean up unused code later on)

Reply:
Addressed in a024633 - I renamed the specific style to searchbox and am now only applying it to the search-box in Variant.vue and Region.vue

I think `.lz-data_layer-tooltip z-index:2` should apply globally. Are there any issues with keeping it in common.css?